sql/src/main/java/io/crate/jobs/PageDownstreamContext.java

```diff
@@ -59,6 +61,7 @@
     public PageDownstreamContext(int id,
                                  String name,
                                  PageDownstream pageDownstream,
+                                 RowReceiver pageDownstreamRowReceiver,
```

Comment:
is the rowReceiver here really required? Seems like now it is either part of the projectorChain or it will be part of the ProjectorChainContext